Common/Wrapper/TraktManager.swift

```diff
@@ -40,6 +40,15 @@ public class TraktManager {
     
     let session: URLSessionProtocol
     
+    // MARK: - Device codes
+    
+    // Would it be better to define a dictionary or a struct?
+    public var deviceCode: String?
+    public var userCode: String?
+    public var verificationURL: String?
+    public var timeInterval: TimeInterval?
+    public var expiresIn: TimeInterval?
```

Comment:
I'll have to look into the API some more before I know if a struct would be better than the variables kept in this class. I would recommend adding more documentation from the Trakt API to define all of these.

Reply:
Yeah, I had looked into that. I wasn't sure either. But I needed them to make the authorization functions working on tvOS.